package.json

```diff
@@ -55,6 +55,7 @@
     "json-stable-stringify": "^1.0.0",
     "levn": "^0.3.0",
     "lodash": "^4.0.0",
+    "minimatch": "^3.0.0",
```

Comment:
I agree that this seems redundant, but `glob` is specifically for matching patterns against files and directories on disk. In this case, `minimatch` is being used to check a file path (which is already confirmed to exist) against a specified pattern, which `glob` is not able to do. And `glob` actually uses `minimatch` under the hood -- it just adds the filesystem traversal logic -- so the results should be the same.